raiden/tests/utils/factories.py

```diff
@@ -501,3 +559,46 @@ def make_transfers_pair(privatekeys, amount, block_number):
         assert channel.is_lock_locked(pay_channel.our_state, UNIT_SECRETHASH)
 
     return channel_map, transfers_pair
+
+
+# ALIASES
+make_channel = make_channel_state
```

Comment:
Why have aliases? I would rather keep 1 function name per factory and use it consistently throughout the tests.

Reply:
backwards compatibility, otherwise tests would fail.

Reply:
Okay I see -- so to make the PR shorter. That makes sense. But we should get rid of them in a subsequent PR and fix the tests.